sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/PlanParserSuite.scala

```diff
@@ -1301,7 +1301,7 @@ class PlanParserSuite extends AnalysisTest {
       "timestamp expression cannot contain subqueries")
   }
 
-  test("PERCENTILE_CONT function") {
+  test("PERCENTILE_CONT & PERCENTILE_DISC") {
     def assertPercentileContPlans(inputSQL: String, expectedExpression: Expression): Unit = {
```

Comment:
nit: rename this function to `assertPercentileBasePlans` ?

Reply:
How about `assertPercentilePlans` ?